### PR TITLE
ci: enable 'release-please' action

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/459/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/459/storybooks/ag-grid-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/459/storybooks/indicators-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/459/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/459/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/459/storybooks/popovers-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/459/storybooks/tabs-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/459/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/459/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/459/playground/)

File: .github/workflows/release-please.yml [hunk 1]
```diff
@@ -11,15 +11,13 @@ jobs:
         id: release
         with:
           release-type: 'node'
-          target-branch: '${{ github.ref_name }}'
+          default-branch: '${{ github.ref_name }}'
```
Comment (Blackbaud-SteveBrush) [inline .github/workflows/release-please.yml:14]: I didn't see an argument for `target-branch` in the config? https://github.com/google-github-actions/release-please-action/blob/main/action.yml

File: .prettierignore [hunk 2]
```diff
@@ -9,3 +9,6 @@ package-lock.json
 
 # Don't format the following since the order of its import statements is deliberate.
 test.ts
+
+# The 'release-please' GitHub action is resposible for the changelog's content.
+CHANGELOG.md
```
Comment (Blackbaud-SteveBrush) [inline .prettierignore:14]: Release Please generates a changelog that does not satisfy our Prettier rules.

File: .github/workflows/release-please.yml (hunk 1 above)
Comment (johnhwhite) [reply .github/workflows/release-please.yml:14]: The CLI renamed this param, but looks like you're right, the action uses `default-branch`.

**Review by johnhwhite — APPROVED** [review]

**Pull request merged**